docs/appkit/shared/options.mdx

```diff
@@ -277,6 +278,25 @@ createAppKit({
 })
 ```
 
+### legalCheckbox
+
+Enable or disable the terms of service and/or privacy policy checkbox.
+
+```ts
+createAppKit({
+  //...
+  features: {
+    legalCheckbox: true
```

Comment:
```suggestion
    legalCheckbox: true
    termsConditionsUrl: '...',
    privacyPolicyUrl: '...'
```

do you think it would be great to add this ?